apps/emqx_resource/src/emqx_resource.erl

```diff
@@ -146,7 +146,7 @@ create(InstId, ResourceType, Config) ->
 -spec create(instance_id(), resource_type(), resource_config(), create_opts()) ->
```

Comment:
Do we need to update the return val spec here since it could return `{retry, _, _,_}` ?

Reply:
This function should handle retry perhaps?

Reply:
@zhongwencool WDYT?